src/txn/txn_recover.c

```diff
@@ -974,8 +1004,12 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
          */
         WT_ERR(session->iface.checkpoint(&session->iface, "force=1"));
 
-    /* Initialize the connection's base write generation after rollback to stable. */
-    WT_ERR(__wt_metadata_init_base_write_gen(session));
+    /*
+     * Rollback to stable may have updated inconsistent transaction ids. Update the connection base
```

Comment:
What is an inconsistent transaction ID? And how does that influence the write generation?

Reply:
Added more details to explain. Those are stale transaction ids.